generators/generator-base.js

```diff
@@ -2085,7 +2085,7 @@ module.exports = class extends PrivateBase {
             this.log(chalk.cyan(`\nKeyStore '${keyStoreFile}' already exists. Leaving unchanged.\n`));
             done();
         } else {
-            shelljs.mkdir('-p', `${SERVER_MAIN_RES_DIR}config/ssl`);
+            shelljs.mkdir('-p', `${SERVER_MAIN_RES_DIR}config/tls`);
```

Comment:
Sorry about that line, that broke the master branch, and I just fixed it, so you'll have a conflict here

Reply:
Rebased in order to remove this commit.